.github/workflows/ci.yml

```diff
@@ -112,3 +125,9 @@ jobs:
           nox -s test-3.11 -- \
             -m integration \
             tests/functional/test_truststore.py
+
+      - name: Run pip debug
+        working-directory: "${{ github.workspace }}/pip"
+        run: |
+          python -m pip install -e .
+          python -m pip debug
```

Comment:
@sethmlarson What's this part for?

Reply:
Maybe we won't need this because we're using the integration test suite, but was seeing failures for `pip debug` on our current vendoring PR. Tbh we can likely remove this since the integration suite will cover it.

Reply:
I'm going to leave it for now, we'll need to change things once we land the truststore vendoring PR anyways too. Can reevaluate then.